pkg/apis/apis/v1alpha1/types.go

```diff
@@ -350,6 +354,18 @@ type Identity struct {
 	SecretRef *corev1.SecretReference `json:"secretRef"`
 }
 
+// APIExportPolicy is a wrapper type around the multiple options that would be allowed.
+type APIExportPolicy struct {
+	//Local is policy that is defined in same namespace as API Export.
```

Comment:
```suggestion
	// local is policy that is defined in same namespace as API Export.
```

API go doc are always about the low-case field names as used in JSON.

Reply:
Done